lib/ddlog_std.rs

```diff
@@ -997,6 +997,16 @@ pub fn map_keys<K: Ord + Clone, V>(map: &Map<K, V>) -> Vec<K> {
     }
 }
 
+pub fn map_values<K: Ord, V: Clone>(map: &Map<K, V>) -> Vec<V> {
+    Vec {
+        vec: map.x.values().cloned().collect(),
+    }
+}
+
+pub fn map_nth<K: Ord, V: Clone>(m: &Map<K, V>, n: &std_usize) -> Option<V> {
+    option2std(m.x.iter().nth(*n as usize).map(|(_, v)| v.clone()))
```

Comment:
I'd be wary of this function since the order of values in a map is non-deterministic

Reply:
We can't have maps with nondet ordering in ddlog because we allow iterating over maps. This is a btreemap.